docker/local/api/config.json

```diff
@@ -3,6 +3,9 @@
     "Human": true,
     "Debug": true
   },
+  "Gateway": {
```

Comment:
Adds this to `local`. Without this config, the `default` points to `https://testnet.tableland.network` which is weird